test/functional/api/cas/casadm.py

```diff
@@ -79,6 +79,20 @@ def try_add(core_device: Device, cache_id: int):
     return Core(core_device.system_path, cache_id)
 
 
+def purge_cache(cache_id: int):
+    output = TestRun.executor.run(script_purge_cache(str(cache_id)))
```

Comment:
I believe run_expect_success could be used

Reply:
All casadm cmd wrappers in this file use `TestRun.executor.run()` so I would prefer to leave it for consistency